library/core/src/main/java/com/google/android/exoplayer2/C.java

```diff
@@ -976,6 +976,54 @@ private C() {}
    */
   public static final int NETWORK_TYPE_OTHER = 8;
 
+  /**
+   * Adaptation set's role descriptor value (ISO 23009-1)
+   */
+  @Documented
+  @Retention(RetentionPolicy.SOURCE)
+  @IntDef({
```

Comment:
As per the issue thread, I'd suggest merging the two IntDefs and making it `flag = true`. You can then also remove the `UNSET` constants, since unset is just 0 for flags.